UnitsNet.Tests/CustomCode/ParseTests.cs

```diff
@@ -164,6 +164,55 @@ public void TryParseLengthUnitUsEnglish(string s, bool expected)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [InlineData("!")]
+        [InlineData("@")]
+        [InlineData("#")]
+        [InlineData("$")]
+        [InlineData("%")]
+        [InlineData("^")]
+        [InlineData("&")]
+        [InlineData("*")]
+        [InlineData("-")]
+        [InlineData("_")]
+        [InlineData("?")]
+        [InlineData("123")]
+        [InlineData("od to")]
+        public void TryParseLengthUnitAbbreviation(string s)
+        {
+            UnitSystem unitSystem = UnitSystem.GetCached("en-US");
+            string abbrev = $"m{s}s";
+            unitSystem.MapUnitToAbbreviation(UnitsNet.Units.LengthUnit.Meter, abbrev);
+            UnitsNet.Units.LengthUnit result;
+            bool actual = unitSystem.TryParse<UnitsNet.Units.LengthUnit>(abbrev, out result);
+            Assert.Equal(true, actual);
+            Assert.Equal(UnitsNet.Units.LengthUnit.Meter, result);
+        }
+
+        [Theory]
+        [InlineData("!")]
+        [InlineData("@")]
+        [InlineData("#")]
+        [InlineData("$")]
+        [InlineData("%")]
+        [InlineData("^")]
+        [InlineData("&")]
+        [InlineData("*")]
+        [InlineData("-")]
+        [InlineData("_")]
+        [InlineData("?")]
+        [InlineData("123")]
+        [InlineData("od to")]
```

Comment:
Ah, indeed you are right. `od to` was simply to test a space and the name was confusing. I will change to "SpecialCharacters".